.travis.yml

```diff
@@ -12,29 +18,22 @@ matrix:
   fast_finish: true
 
   include:
-    - php: 7.1
+    - php: 7.2
       env: PREFER_LOWEST=1
 
-    - php: 7.2
+    - php: 7.4
       env: CHECKS=1 DEFAULT=0
 
 before_install:
   - phpenv config-rm xdebug.ini
 
-before_script:
-  - if [[ $PREFER_LOWEST != 1 ]]; then composer install --prefer-source --no-interaction; fi
-  - if [[ $PREFER_LOWEST == 1 ]]; then composer lowest-setup; fi
+install:
+  - if [[ $PREFER_LOWEST != 1 ]]; then composer install --prefer-dist --no-interaction; fi
+  - if [[ $PREFER_LOWEST == 1 ]]; then composer update --prefer-lowest --prefer-stable --no-interaction; fi
 
 script:
   - if [[ $DEFAULT == 1 ]]; then composer test; fi
-
-  - if [[ $PREFER_LOWEST == 1 ]]; then composer lowest; fi
-
   - if [[ $CHECKS == 1 ]]; then composer cs-check; fi
 
-cache:
```

Comment:
I would have kept this below, as this is just noise at the top

Reply:
I just aligned it with cakephp.

Reply:
in cake i would have done the same rather, moving irrelevant things to the bottom

Reply:
as you like to say "feel free to change it if you want" :)